cmd/db_migrations.go

```diff
@@ -59,38 +60,43 @@ Error: %s`,
 	}
 
 	if latestVersionInDB.version > latestDBVersion {
-		die(`Looks like you downgraded omm. You should either delete omm's
+		return fmt.Errorf(`Looks like you downgraded omm. You should either delete omm's
```

Comment:
```suggestion
		return errors.New(`Looks like you downgraded omm. You should either delete omm's
```

You are not formating here